cmd/kubeadm/app/util/jsonstanza/jsonstanza.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
This file still looks verbatim from https://github.com/mholt/caddy/blob/master/caddyfile/json.go which is not by "The "Kubernetes Authors". You _are_ planning to preserve our original copyright, I imagine?